charts/outdated/templates/service.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     app.kubernetes.io/component: api
 spec:
   type: {{ .Values.api.service.type }}
+  publishNotReadyAddresses: true
```

Comment:
are you sure you need this? setting it to true will make the service send requests to the service that might fail

Reply:
It shouldn't be necessary, without it i couldn't get it to run locally.

Reply:
its gone now and stuff works